app/views/fae/application/_content_form.html.slim

```diff
@@ -6,6 +6,9 @@ ruby:
   markdown           ||= false
   markdown_supported ||= false
   hint               ||= nil
+  wrapper_html       ||= {}
+  input_options      ||= {}
+  input_html         ||= {}
```

Comment:
Can you add this local as a method argument for `fae_content_form` and document or remove it here?